Gems/Atom/Tools/MaterialCanvas/Assets/MaterialCanvas/GraphData/Nodes/MaterialOutputs/BasePBR/MaterialGraphName_LowEndForward.shader

```diff
@@ -1,10 +1,4 @@
 {
-    // Note: "LowEnd" shaders are for supporting the low end pipeline
-    // These shaders can be safely added to materials without incurring additional runtime draw
-    // items as draw items for shaders are only created if the scene has a pass with a matching
-    // DrawListTag. If your pipeline doesn't have a "lowEndForward" DrawListTag, no draw items
-    // for this shader will be created.
-
```

Comment:
why remove this comment?

Reply:
The comments were copied from and still exist in the original files for each of these material types and will change once the material pipeline branch is integrated.